lib/main.dart

```diff
@@ -0,0 +1,193 @@
+
+class _HomePageState extends State<HomePage> {
+  int _likeCounter = 0;
+
+  void addLike(){
+    setState(() {
+      _likeCounter++;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: Center(
+          child: ListView(
+              children: [
+                Image.asset(
+                    'assets/images/ysera_card.png',
+                  height: 500,
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    const Icon(
+                      Icons.favorite,
+                      color: Colors.pink,
+                      size: 30,
+                    ),
+                    const SizedBox(
+                      width: 10,
+                    ),
+                    Text(
+                      "$_likeCounter",
+                      style: const TextStyle(
+                          color: Colors.black,
+                          fontSize: 30
+                      ),
+                    )
+                  ],
+                ),
+                IntrinsicHeight(
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: [
+                        Column(
+                          crossAxisAlignment: CrossAxisAlignment.center,
+                          children: const [
+                            Padding(
+                              padding: EdgeInsets.all(8.0),
+                              child: Text(
+                                  "Name: Ysera",
+                                style: TextStyle(
+                                  color: Colors.black,
+                                  fontSize: 24,
+                                ),
+                                textAlign: TextAlign.left,
+                              ),
+                            ),
+                            Padding(
+                              padding: EdgeInsets.all(8.0),
+                              child: Text(
+                                "Type: Minion",
+                                style: TextStyle(
+                                  color: Colors.black,
+                                  fontSize: 24,
+                                ),
+                                textAlign: TextAlign.left,
+                              ),
+                            ),
+                            Padding(
+                              padding: EdgeInsets.all(8.0),
+                              child: Text(
+                                "Faction: Neutral",
+                                style: TextStyle(
+                                  color: Colors.black,
+                                  fontSize: 24,
+                                ),
+                                textAlign: TextAlign.left,
+                              ),
+                            ),
```

Comment:
this whole widget can be abstracted to no repeat code, also the values can be constants

Reply:
Done